.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: npm publish
+
+on:
+  workflow_dispatch:
+
+jobs:
+  npm-publish:
+    name: Publish npm package
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust 1.81 # https://github.com/estie-inc/wasm-xlsxwriter/blob/a2696f5e9e729dfe08798309e777cc0e1f3d285b/README.md#browser-support
+        run: rustup install 1.81.0 && rustup default 1.81.0
```

Comment:
[dtolnay/rust-toolchain](https://github.com/dtolnay/rust-toolchain/) is widely used to pin a specific Rust version in GitHub Actions.